src/pylogctx/django.py

```diff
@@ -4,19 +4,19 @@
 
 from django.conf import settings
 
-from pylogctx import context
+from pylogctx import context, AdapterNotFound
 
 
 logger = logging.getLogger(__name__)
 
 
-class ExtractRequestContextMiddleware(object):
+class OuterMiddleware(object):
     def process_request(self, request):
-        extractor = settings.PYLOGCTX_REQUEST_EXTRACTOR
+        context.clear()
         try:
-            context.update(**extractor(request))
-        except Exception:
-            logger.exception()
+            context.update(request)
+        except AdapterNotFound:
+            logger.debug("Can't adapt %s for log.", request.__class__.__name__)
```

Comment:
I think that debug level is too low. In case of misconfiguration this error will silently pass (nobody runs with debug mode in production and only few reading debug logs in testing environments).